src/utility/serializable.hpp

```
@@ -0,0 +1,169 @@
+
+            std::apply([&](const auto&... meta) { (print_one(meta), ...); }, metas);
+
+            return result;
+        }
+    };
+
+}
+
+struct Serializable {
+    template <typename Metas, std::size_t... Idx>
+    constexpr auto make_serializable_impl(Metas metas, std::index_sequence<Idx...>) {
```

Clang (14.0.6)

[warning] all parameters should be named in a function

(readability-named-parameter)